t5x/contrib/gpu/t5/layers.py

```diff
@@ -300,16 +305,25 @@ def __call__(self,
       dropout_rng = self.make_rng('dropout')
 
     # Apply attention.
-    x = dot_product_attention(
+    if os.environ.get("CUDNN_FLASH_ATTENTION") == "true":
+      x = cudnn_dot_product_attention(
         query,
         key,
         value,
-        bias=attention_bias,
-        dropout_rng=dropout_rng,
+        attention_bias,
```

Comment:
nit: would be nice to maintain the same kwarg syntax: `bias=attention_bias` to minimize diff hunk